radio/apps/schedules/admin.py

```diff
@@ -97,8 +94,8 @@ def get_urls(self):
             'app_name': self.model._meta.app_label,
             'model_name': self.model._meta.model_name,
         }
-        custom_urls = patterns('',
-            url(
+        custom_urls = patterns(
+            '', url(
```

Comment:
this was mangled by the pep8 tool (continuation line under-indented for visual indent)